src/components/Oneof.vue

```diff
@@ -166,10 +166,9 @@ const currentIns = getCurrentInstance()
  * form validation after setting modelValue to null following enable.
  */
 watch(isDisabled, async (val) => {
-  if (!val) {
+  if (val) {
     const formItem = props.formItem || currentIns?.parent?.exposed
     if (isFunction(formItem?.clearValidate)) {
-      await nextTick()
```

Comment:
Removing here may cause a little problem. Deleting here is to solve the https://emqx.atlassian.net/browse/ED-753, right? Let's take a look when I finish processing the webhook😵